sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1236,7 +1236,7 @@ object SQLConf {
   val STATE_STORE_FORMAT_VALIDATION_ENABLED =
     buildConf("spark.sql.streaming.stateStore.formatValidation.enabled")
       .internal()
-      .doc("When true, check if the UnsafeRow from the state store is valid or not when running " +
+      .doc("When true, check if the checkpoint from the state store is valid or not when running " +
```

Comment:
`checkpoint from the state store` looks a bit weird. `data from the state store`?

Reply:
Thanks, fix done